src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.EntityMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.OverseasEntityDueDiligenceMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.PresenterMock;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.EntityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class OverseasEntitySubmissionDtoValidatorTest {
+
+    private static final String CONTEXT = "12345";
+    private OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator;
+    @Mock
+    private AddressDtoValidator addressDtoValidator;
+    @Mock
+    private EntityDtoValidator entityDtoValidator;
+    @Mock
+    private PresenterDtoValidator presenterDtoValidator;
+    @Mock
+    private OverseasEntityDueDiligenceValidator overseasEntityDueDiligenceValidator;
+    @Mock
+    private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+
+    @Test
+    void testOverseasEntityDueDiligenceValidator() {
+        overseasEntitySubmissionDtoValidator = new OverseasEntitySubmissionDtoValidator(
```

Comment:
you should be able to replace the manual creation of the OverseasEntitySubmissionDtoValidator with an 
`@InjectMocks
    private OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator; `

Reply:
Changed this to inject mocks